### PR TITLE
Introduce updated_at column to policy db

### PR DESCRIPTION
**Pull request by pradyutsarma**
1. Introduce the column in db changelog and sequelize model
2. Trigger value on insert/update of row
3. Requisite tests

### CODE REVIEW

File: api/test/unit/models/policy.test.js [hunk 1]
```diff
@@ -11,23 +12,57 @@ describe('Create a Policy', function() {
...
+		        expect(error).to.have.deep.property('message').equal('Validation error');
+		        done();
+		    });
+	    });
+	  });	  
   });
+  
+  context ('Test create and update policies with updated_at checks', function () {
+	  beforeEach(function() {
+		policy.truncate();
```
Comment (fraenkel) [inline api/test/unit/models/policy.test.js]: missing return

File: api/test/unit/models/policy.test.js [hunk 2]
```diff
@@ -11,23 +12,57 @@ describe('Create a Policy', function() {
     fakePolicy = JSON.parse(fs.readFileSync(__dirname+'/../fakePolicy.json', 'utf8'));
   });
 
-  beforeEach(function() {
-    policy.truncate();
-  })
+  context ('Test create policies', function () {
+	  beforeEach(function() {
+		policy.truncate();
```
Comment (fraenkel) [inline api/test/unit/models/policy.test.js]: missing return

**Comment by fraenkel** [comment]
Is there a separate story to flow this as some type of etag or something so other components have some idea of what "version" of data they are using?

File: api/test/unit/models/policy.test.js (hunk 2 above)
Comment (pradyutsarma) [inline api/test/unit/models/policy.test.js]: Fixed.

**Comment by pradyutsarma** [comment]
@fraenkel we discussed this in the last call. Maybe we can introduce some type of etag in the GET API response header generated out of the attributes including the updated_at data. But then, will the other components query for this per app?

**Comment by fraenkel** [comment]
We need something. But more importantly, is the usage for scheduler and other components which will use it to determine if the underlying policy has changed/replaced.
We do need something to track this.

**Comment by pradyutsarma** [comment]
I will create a story. Let's discuss there.

**Comment by fraenkel** [comment]
Merged

**Pull request closed without merging**